Name: 4G Icon Instead of 3G
Version: 2.2.0-1
Author: Jason Robitaille
Description: Replaces the 3G top bar icon with the 4G one  (MIT license)

--- a/usr/lib/luna/system/luna-systemui/stylesheets/systemui.css
+++ b/usr/lib/luna/system/luna-systemui/stylesheets/systemui.css
@@ -215,8 +215,8 @@ body.palm-default {
 #datanet.connected-evdo3g { background-image: url(../images/network-3g-connected.png); }
 #datanet.dormant-evdo3g { background-image: url(../images/network-3g-dormant.png); }
 #datanet.connected-gprs { background-image: url(../images/network-gprs-connected.png); }
-#datanet.connected-umts { background-image: url(../images/network-3g-connected.png); }
-#datanet.connected-hsdpa { background-image: url(../images/network-3g-connected.png); }
+#datanet.connected-umts { background-image: url(../images/network-4g-connected.png); }
+#datanet.connected-hsdpa { background-image: url(../images/network-4g-connected.png); }
 #datanet.connected-hspa-4g { background-image: url(../images/network-hsdpa-plus-connected.png); }
 
 /* Ev and 1x dual signal strength */
